src/modules/previewpane/common/controls/FormHandlerControl.cs

```diff
@@ -111,6 +111,11 @@ public virtual void Unload()
 
                 this.Controls.Clear();
             });
+
+            // Call garbage collection at the time of unloading of Preview. This is to mitigate issue with WebBrowser Control not able to dispose properly.

```

Comment:
Can you provide a link to the git hub issue?  This way someone in a different fork or branch could check on the status, and find the commits associated with the fix

Reply:
Added a git hub issue link with PR and also in comments.